Makefile

```diff
@@ -18,14 +18,23 @@ build: ## Build the project
 run: build ## Run the 15-puzzle isomorphic app
 	PORT=$(PORT) npm run start
 
+run-cache: build ## Run the 15-puzzle isomorphic app with cache
```

Comment:
Instead of having multiple entry points, it make it easier to document and to maintain to have only one (and use env vars for "feature" flagging)